cl/citations/tasks.py

```diff
@@ -205,10 +110,13 @@ def find_citations_for_opinion_by_pks(
             if matched_opinion.pk not in all_cited_opinions:
                 opinion_ids_to_update.add(matched_opinion.pk)
 
+        # Generate the citing opinion's new HTML with inline citation links
+        opinion.html_with_citations = create_cited_html(opinion, citations)
```

Comment:
It seemed to me that this shouldn't actually be in the transaction block at all.